### PR TITLE
Add Craft CMS

### PR DESCRIPTION
**Pull request by chasegiunta**
https://github.com/chasegiunta/awesome-craft

Curated list of awesome Craft CMS plugins, technical articles/tutorials, and praise articles (sought by many developers promote & compare the platform). Craft is a CMS that’s laser-focused on doing one thing really, really well: _managing content_. Built to be as flexible as possible, without compromising on the ease of use for content authors.

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Has the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- Each link description starts with an uppercase character and ends with a period.<br>Example: `- [AVA](…) - JavaScript test runner.`
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by chasegiunta** [comment]
Reluctantly pinging @sindresorhus - sorry!

**Comment by chasegiunta** [comment]
@sindresorhus It's been almost 30 days since this PR was submitted. Over 2 months since this list was created. A good few hours of work put into this list. You've actively given your feedback on far newer PRs. Please review so I can get to fixing whatever issues need to be addressed and it can be merged. Thank you.

**Comment by legodude17** [comment]
- Describe what articles are about
- Ensure that no descriptions start with `A` or `An`
- Ensure that every description ends in a period
- Describe the following more fully:
  - Crisp
  - Chatra
  - Olark
  - Drift

**Comment by chasegiunta** [comment]
@legodude17 I really appreciate the review! Good eye. I've addressed 3 of the 4 points you listed:

- Ensure that no descriptions start with `A` or `An` https://github.com/chasegiunta/awesome-craft/commit/43a4385553f5cc0161057d57da3255022f943b47
- Ensure that every description ends in a period https://github.com/chasegiunta/awesome-craft/commit/3b69316a703ab33083ae665abb2fddd867b47fb5
- Described Crisp, Chatra, Olark, and Drift service plugins more fully https://github.com/chasegiunta/awesome-craft/commit/a08d2d53e36a5fb3c90004a54c8c81f359031fac

As for the article descriptions, the titles _are_ descriptions themselves. I believe if I were to try & write _additional_ descriptions, they would be redundant. I hope this practice is okay, as the lists that @sindresorhus maintains himself (and others I've seens) also follow the same format

- https://github.com/sindresorhus/awesome-electron#articles
- https://github.com/sindresorhus/awesome-nodejs#articles
- https://github.com/sindresorhus/awesome-npm#articles

**Comment by legodude17** [comment]
@chasegiunta Yeah, the articles do describe themselves.

File: readme.md [hunk 1]
```diff
@@ -408,6 +408,7 @@
 - [Wagtail](https://github.com/springload/awesome-wagtail) - Django CMS focused on flexibility and user experience.
 - [Textpattern](https://github.com/drmonkeyninja/awesome-textpattern) - Lightweight PHP-based CMS.
 - [Drupal](https://github.com/nirgn975/awesome-drupal) - Extensible PHP-based CMS.
+- [Craft CMS](https://github.com/chasegiunta/awesome-craft) - Content-first CMS
```
Comment (sindresorhus) [inline readme.md]: You're missing a `.` at the end.

**Comment by sindresorhus** [comment]
@chasegiunta Please understand that I have a life outside of this project and open source. It's summer and I've been busy in real life. I have a lot of tabs open with things I need to do, and I don't always do them in order. You have spent a few hours creating your list, but I have spent hundreds of hours on this project.

**Comment by sindresorhus** [comment]
Back to the review:

- List looks pretty good. Nice work on the header!
- Links and descriptions should be separated by a `-`. For example `[Title](URL) - Description.` (Also note the uppercased first character in the description and it ending in a dot).
- I'd like to see some links to community resources: Stack Overflow tag, IRC, chat, forums, communities, Reddit, etc. Example: https://github.com/sindresorhus/awesome-electron#community

**Comment by chasegiunta** [comment]
@sindresorhus You're right. My frustration was mainly due to some newer PRs getting attention while this one remained unseen, but nevertheless in the scope of things, my impatience is uncalled for. I appreciate you making time for it!

Made the following changes:

- I've edited the PR & added a dot to the end of the description (cannot believe I missed that). https://github.com/sindresorhus/awesome/pull/990/commits/e02a6d209521f38fab482a569e1487c9ef818cb7
- Links and descriptions should be separated by a '-' https://github.com/chasegiunta/awesome-craft/commit/a892cb6114c48e0120b8cbf08dd7764e90c3b155

So, I _only didn't_ apply the above format (dash to separate, uppercased first character, ending in dot) to the **articles**. `Article Title - By John Doe.` doesn't quite sit right with my grammatical brain, yet I think _it is_ beneficial to give the author & agency credit in the listing. If this is okay, great! If not, I'll just remove the descriptive credits.

- Community resources https://github.com/chasegiunta/awesome-craft/commit/bff97e9478d0e1091213fbfa86048e9c98d10c99

I did have "Third Party" resources listed at the very bottom, but "Community" better fits that section and encompasses official resources. Added: official Slack group, Twitter, and Facebook Links. There is an unofficial Craft subreddit, but it's not very active, and isn't an _awesome_ resource imo.

**Comment by sindresorhus** [comment]
👍

**Pull request merged**